Apps/APN-UIKit/APN UIKit/Router/LoginRouter.swift

```diff
@@ -4,7 +4,7 @@ import UIKit
 protocol LoginRouting {
     func routeToDashboard()
     func routeToSettings()
```

Comment:
wonder if this is necessary? 
we now have `routeToSettings(_ withInfo: [String: String]?)` with optional info? 

I believe both of them take it to same screen.

Reply:
Ah! Yes... good catch! Thanks Shahroz. I've fixed it.